### ISSUE TITLE
Test errors with pytest==7.2.0

### ISSUE THREAD

**Issue by whyscream** [stephrdev/pytest-isort#45]
Just found out that [pytest-isort](https://github.com/stephrdev/pytest-isort) is using `py.io` in https://github.com/stephrdev/pytest-isort/blob/a36c16ecbf454cc25f8564aa39fe652808f56d3f/pytest_isort/__init__.py#L207. For some reason the `pytest-isort` package is not depending on the `py` package, it's depending on pytest to do that. 

With the latest pytest release (7.2.0), pytest has dropped the `py` dependency, and in stead vendored the few parts that they use. See https://github.com/pytest-dev/pytest/issues/10396 and https://docs.pytest.org/en/stable/changelog.html#pytest-7-2-0-2022-10-23 for details.

In short, now all isort tests fail with `FAILED mypackage/__init__.py::ISORT - AttributeError: module 'py' has no attribute 'io'`

**Comment by stephrdev** [comment]
Instead of adding py as dependency I replaced it using the capture helper from pytest. With that, I also dropped Python 3.6 support. A review on this would be great: https://github.com/stephrdev/pytest-isort/pull/47
